src/contracts/core/StrategyManager.sol

```diff
@@ -183,9 +179,9 @@ contract StrategyManager is
      */
     function recordBeaconChainETHBalanceUpdate(address podOwner, uint256 beaconChainETHStrategyIndex, int256 sharesDelta)
```

Comment:
Hmm this stuff is no longer there in SM, I think master needs to be merged into this branch